compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -101,7 +101,7 @@ static void getTensorCoreConfig(
   // Tile sizes are skewed towards small matmul for now. Long term the plan is
   // to not rely on hardcoded configurations.
   if (isFp16) {
-    tileSizes.push_back(TileWorkgroupSizePair({{32, 32, 32}, {64, 2, 1}}));
```

Comment:
I don't think we want to change tile size in this PR. If we need this please make it a separate PR so that we could easily figure out if this causes changes and keep the older configuration as a fall back.